azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerView.kt

```diff
@@ -68,13 +68,13 @@ internal class BannerView : ConstraintLayout {
                 BannerInfoType.RECORDING_STOPPED_STILL_TRANSCRIBING,
                 BannerInfoType.TRANSCRIPTION_STOPPED_STILL_RECORDING,
                 -> {
-                    context.getString(R.string.azure_communication_ui_calling_view_link_privacy_policy_url)
+                    getLocalizedString(R.string.azure_communication_ui_calling_view_link_privacy_policy_url)
```

Comment:
Shouldn't we pass the context also with the hash int of resource string id?
As the getLocalizedString signature is 'getLocalizedString(context: Context, stringKey: Int)'

Reply:
getLocalizedString() have the other reuse method end of this class

Reply:
I see. at line 129. 
This way is much cleaner. Good one. Thanks